### DISCUSSION TITLE
feature request

### DISCUSSION THREAD

**Discussion by xiaofanshifu** [AdguardTeam/AdGuardHome#4604] [Ideas]
It is hoped that DNS rewriting can support adding multiple types of records.
![image](https://user-images.githubusercontent.com/57950591/170830552-1f45556f-608f-47a4-ae4b-904f7bc3d4e7.png)

Don't use dnsrewrite rules.
![image](https://user-images.githubusercontent.com/57950591/170830711-cb7618f1-fed4-432a-8740-7bc844e759bc.png)

For example [synology](https://kb.synology.com/en-us/DSM/tutorial/How_to_set_up_your_domain_with_Synology_DNS_Server)
![image](https://user-images.githubusercontent.com/57950591/170830643-922aca31-5234-4496-a337-b75581c7fab6.png)

**Comment by ainar-g** [comment]
A better way to add records is through the [`dnsrewrite` rules][1].  The current GUI DNS rewrite functionality has many issues and will be reworked in future versions.

[1]: https://github.com/AdguardTeam/AdGuardHome/wiki/Hosts-Blocklists#dnsrewrite

**Reply by ghost** [reply]
目前有计划在未来第几个版本改进这个 dns 重写的功能吗？

**Reply by ghost** [reply]
@ainar-g
